datasets/gov/vmi/mm_registras.csv

```diff
@@ -0,0 +1,45 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/mm_registras,,,,,,,,,,,,"MOKESČIŲ MOKĖTOJŲ (JURIDINIŲ ASMENŲ) KODŲ, TIPŲ, GRUPIŲ DUOMENYS","MOKESČIŲ MOKĖTOJŲ (JURIDINIŲ ASMENŲ) KODŲ, TIPŲ, GRUPIŲ DUOMENYS"
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,MokesciuMoketojas,,,eiles_nr,,,4,,,"MOKESČIŲ MOKĖTOJŲ (JURIDINIŲ ASMENŲ) KODŲ, TIPŲ, GRUPIŲ DUOMENYS","MOKESČIŲ MOKĖTOJŲ (JURIDINIŲ ASMENŲ) KODŲ, TIPŲ, GRUPIŲ DUOMENYS"
+,,,,,eiles_nr,integer,,,,4,open,,Duomenų rinkinio eilės numeris,
+,,,,,klnt_tipas,string,,,,4,open,,Kliento tipas. ,
```

Comment:
```suggestion
,,,,,klnt_tipas,string,,,,3,open,,Kliento tipas. ,
```
1. Iš pateiktų duomenų LJA ir UJA neaišku kas tai per duomenys, rekomenduojama `decription` pateikti paaiškinimus arba nurodyti `enum`
2. Ar šis `property` neturi sąsajų su kitais duomenimis